sdk/samples/azidentity/SDKV1Adapter/example_SDKV1_test.go

```diff
@@ -5,139 +5,70 @@ package azidentitysamples
+	// specify the Azure Resource Manager scope for that cloud in azidext.DefaultAzureCredentialOptions
 	a, err := azidext.NewDefaultAzureCredentialAdapter(nil)
 	if err != nil {
-		panic("failed to get credential")
+		panic(err)
 	}
-	groupsClient.Authorizer = a
-	// use the groups client with the azidentity credential in the authorizer
-	group, err := groupsClient.CreateOrUpdate(
-		context.Background(),
-		groupName,
-		resources.Group{
-			Location: to.StringPtr(location),
-		})
+
+	client := subscription.NewSubscriptionsClient()
+	client.Authorizer = a
+
+	subscriptions, err := client.List(context.Background())
 	if err != nil {
 		panic(err)
 	}
-	fmt.Println(*group.Name)
+	subNames := []string{}
+	for _, s := range subscriptions.Values() {
+		subNames = append(subNames, *s.DisplayName)
+	}
 	// Output:
-	// samplegroup
 }
 
-// ExampleNewClientSecretCredential for using the ClientSecretCredential with the NewAzureIdentityCredentialAdapter and assigning the credential to the
-// SDK V1 authorizer.
-// NewAzureIdentityCredentialAdapter can take any credential type defined in azidentity and convert it to an authorizer that is compatible with the Azure SDK for Go
-// V1 implementation. For a list of the credentials that azidentity includes, please see: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/azidentity.
-// NewClientSecretCredential can be used in place of auth.NewClientCredentialsConfig().
+// ExampleNewClientSecretCredential demonstrates NewAzureIdentityCredentialAdapter, which adapts any azidentity credential type for use with the Azure SDK for Go V1.
 func ExampleNewClientSecretCredential() {
-	groupsClient := resources.NewGroupsClient(subscriptionID)
-	// instantiate a new ClientSecretCredential as specified in the documentation
 	cred, err := azidentity.NewClientSecretCredential(tenantID, clientID, clientSecret, nil)
 	if err != nil {
 		panic(err)
 	}
-	// call azidext.NewAzureIdentityCredentialAdapter with the azidentity credential and necessary scope
-	// NOTE: Scopes define the set of resources and permissions that the credential will have assigned to it.
-	// 		 To read more about scopes, see: https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent
-	a := azidext.NewAzureIdentityCredentialAdapter(
-		cred,
-		azcore.AuthenticationPolicyOptions{
-			Options: azcore.TokenRequestOptions{
-				Scopes: []string{"https://management.azure.com/.default"}}})
-	if err != nil {
-		panic("failed to get credential")
-	}
-	// assign the authorizer to your client's authorizer
-	groupsClient.Authorizer = a
-	// perform an operation with the complete client
-	list, err := groupsClient.ListComplete(context.Background(), "", nil)
+	// This scope is correct for Azure Resource Manager in Azure Public Cloud. Other clouds require a different scope.
+	a := azidext.NewTokenCredentialAdapter(cred, []string{"https://management.azure.com//.default"})
```

Comment:
should it be `https://management.azure.com/.default`

Reply:
That might work as well, but no. The v1 resource was `https://management.azure.com/` so the equivalent v2 scope is `https://management.azure.com//.default` ([docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#trailing-slash-and-default))